src/backend/distributed/deparser/deparse_type_stmts.c

```diff
@@ -388,7 +392,7 @@ AppendColumnDef(StringInfo str, ColumnDef *columnDef)
 		appendStringInfo(str, "%s ", quote_identifier(columnDef->colname));
 	}
 
-	appendStringInfo(str, "%s", format_type_be_qualified(typeOid));
+	appendStringInfo(str, "%s", format_type_extended(typeOid, typmod, formatFlags));
```

Comment:
I need to look at the original PR and discussion why we went for `format_type_be_qualified`. There was a very specific reason IIRC, but not supporting varchar is not good either.

Given there are no tests failing I guess replacing it like this is ok, but need to have a look why we used this in the first place.

Reply:
Ok